charts/fybrik/files/taxonomy/fybrik_module.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "fybrik_module.json",
+    "title": "FybrikModule taxonomy",
+    "type": "object",
+    "description": "Values contributed to by Fybrik modules.",
+    "properties": {
+        "spec": {
+            "type": "object",
+            "properties": {
+                "pluginType": {
+                    "$ref": "taxonomy.json#/definitions/PluginType"
+                },
+                "capabilities": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+		                    "capability": {
+                                "$ref": "taxonomy.json#/definitions/Capability"
+                            },
+                            "api": {
+                                "type": "object",
+                                "properties": {
+                                    "interfaceDetails": {
+                                        "$ref": "taxonomy.json#/definitions/Interface"
+                                    }
+                                },
+                                "required": ["interfaceDetails"]
+                            },
+                            "supportedInterfaces": {
+                                "type": "array",
+                                "items": {
+                                    "type": "object",
+                                    "oneOf": [
+                                        {
+                                            "source": {
+                                                "$ref": "taxonomy.json#/definitions/Interface"
+                                            }
+                                        },
+                                        {
+                                            "sink": {
+                                                "$ref": "taxonomy.json#/definitions/Interface"
+                                            }
+                                        }
+                                    ]
+                                }
+                            },
+                            "actions": {
+                                "type": "array",
+                                "items": {
+                                    "type": "object",
+                                    "properties": {
+                                        "id": {
+                                            "$ref": "taxonomy.json#/definitions/Action"
```

Comment:
`taxonomy.json#/definitions/Action` is an object with a required `name` field (and optionally additional fields). It's not the equivalent of `id`. I think that we must change the FybrikModule definition to match the taxonomy Action structure (instead of ID and level).

Reply:
@roee88 @ronenkat So we need to change the FybrikModule CRD structure?